### PR TITLE
docs - question details sidebar

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
Updates the sharing answers page to include:

- official collections.
- new question detail sidebar
- promotion of some headings

### CODE REVIEW

File: docs/users-guide/06-sharing-answers.md [hunk 1]
```diff
@@ -1,77 +1,70 @@
...
 
-#### Your personal collection
+- **Edit details** (Pencil icon). Change the title of the question, and add some description for context. Adding a description will also make the question easier to find using the search bar. You can also select more options to [cache the results of the question](#caching-results).
+- **Add to dashbboard** (Dashboard icon with plus symbol). See [dashboards][dashboards].
+- **Move** (Document icon with right arrow). Relocate the question to a different [collection](#collections).
+- **Duplicate** (Square with little square). Create a copy of the question. Keep in mind that whenever you start editing a saved question, Metabase will create a copy of the question. You can either save your edits as a new question, or overwrite the original saved question.
+- **Archive** (Folder with down arrow). See [Archiving items](#archiving-items).
 
-In addition to the collections you and your teammates have made, you'll also always have your own personal collection that only you and administrators can see. To find it, click on the "browse all items" button on the homepage and click on "my personal collection" in the list of collections.
+### Caching results
```
Comment (mazameli) [inline docs/users-guide/06-sharing-answers.md:29]: I forget: is per-question cache settings a paid feature?

File: docs/users-guide/06-sharing-answers.md [hunk 2]
```diff
@@ -1,77 +1,70 @@
...
+If your results don't change frequently, you may want to cache your results, that is: store your results in Metabase so that the next time you visit the question, Metabase can retrieve the stored results rather than query the database again. For example, if your data only updates once a day, there's no point in querying the database more than once a day, as they data won't have changed. Returning cached results can be significantly faster, as the database won't have to redo the work to answer your query.
 
-#### Pinned items
+To cache results, click on the down arrow next to the question's title to open the __Question detail sidebar__, then click on the __Pencil icon__ to __Edit details__. In the Modal that pops up, in the bottom left, select __More options__. There you'll be able to tell Metabase how long it should cache the question's results. This caching will only apply to this specific question; admins can [configure database-wide caching settings][caching] in the __Admin panel__.
```
Comment (mazameli) [inline docs/users-guide/06-sharing-answers.md:33]: I think we need to mention that you can still set global caching, and that doing it here is a way to override that global cache for a specific question that might have a different natural cadence to it.

File: docs/users-guide/06-sharing-answers.md [hunk 3]
```diff
@@ -1,77 +1,70 @@
...
 
-![Pins](images/sharing-answers/pinned-items.png)
+### Question moderation
 
-In each collection, you can pin important or useful dashboards or questions to make them stick to the top of the screen. Pinned items will also be displayed as large cards to make them stand out well. If you have Curate permissions for a collection, you can pin and un-pin things, and drag and drop pins to change their order.
+Administrators can **Verify** a question by clicking on the **Verify checkmark** in the **Moderation** section of the **Question detail sidebar**. Verifying a question is a simple way for an administrator to signal that they've reviewed the question and deemed it to be trustworthy. That is: the question is filtering the right columns, or summarizing the right metrics, and querying records from the right tables. Once verified, the question will have a verified icon next to the question's title.
```
Comment (mazameli) [inline docs/users-guide/06-sharing-answers.md:37]: Two things:
- I believe Verification is a paid feature — are we calling that out?
- I'd suggest mentioning that Verified items are more likely to show up higher in search suggestions and search results.

File: docs/users-guide/collections.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+### Moving items from collection to collection
+
+To move a question, dashboard, or pulse into a collection, or from one collection to another, just click and drag it onto the collection where you want it to go. You can also click on the `…` menu to the right of the question and pick the Move action. If you're trying to move several things at once, click on the items' icons to select them, then click the Move action that pops up at the bottom of the screen.
+
+![Selecting questions](images/collections/question-checkbox.png)
+
+Note that you have to have Curate permission for the collection that you're moving a question into _and_ the collection you're moving the question out of.
+
+### Search
```
Comment (mazameli) [inline docs/users-guide/collections.md:46]: Not sure that I understand why Search is found within the doc on collections.

**Review by flamber — APPROVED** [review]

**Pull request merged**